cardano-api/src/Cardano/Api/IPC/Monad.hs

```diff
@@ -0,0 +1,65 @@
+{-# LANGUAGE DerivingVia #-}
+
+module Cardano.Api.IPC.Monad
+  ( LocalStateQueryScript
+  , sendMsgQuery
+  , setupLocalStateQueryScript
+  ) where
+
+import Cardano.Api.Block
+import Cardano.Api.IPC
+import Control.Applicative
+import Control.Concurrent.STM
+import Control.Monad
+import Control.Monad.IO.Class
+import Control.Monad.Trans.Cont
+import Data.Either
+import Data.Function
+import Data.Maybe
+import Data.Ord
+import Shelley.Spec.Ledger.Scripts ()
+import System.IO
+
+import qualified Ouroboros.Network.Protocol.LocalStateQuery.Client as Net.Query
+import qualified Ouroboros.Network.Protocol.LocalStateQuery.Type as Net.Query
+
+newtype LocalStateQueryScript block point query r m a = LocalStateQueryScript
```

Comment:
Can we add a comment here with a short description? We should indent at `via` as well.

Reply:
Added comments.